app/Models/Forum/Topic.php

```diff
@@ -425,4 +426,30 @@ public function deleteWithCover()
 
         $this->delete();
     }
+
+    public function isADoublePost(User $user)
+    {
+        if ($user === null) {
+            return false;
+        }
+        if ($user->user_id === $this->topic_poster) {
```

Comment:
this conditional should just assign `$minimumHours` and do the actual check afterward.